### ISSUE TITLE
Done and later buttons on details dialog 

### ISSUE THREAD

**Issue by manparvesh** [IdeaTrackerPlus/IdeaTrackerPlus#17]
Description:
-------------------
Done and later buttons should be displayed on details dialog of an idea too

**Comment by ghost** [comment]
Hello! I would like to take up this issue!

**Comment by manparvesh** [comment]
@sangeethakp sure!

**Comment by ghost** [comment]
Hi Manparvesh, 
I would like to clarify if this is the edit dialog you have specified here?
![2017-09-30 17 21 05](https://user-images.githubusercontent.com/6271286/31045556-cd4ceffe-a603-11e7-9796-0a2a7db02558.jpg)

**Comment by manparvesh** [comment]
Hi @sangeethakp sorry for the confusion. I have updated the title and description.
I meant that there should be buttons in the details dialog of the idea too. The dialog that pops up when you tap on an idea.

**Comment by ghost** [comment]
Hi @manparvesh , clarifying this, we need to have Done and Later buttons on the dialog that pops up when we click on an idea in the "IDEAS" Tab right?

Clicking on "Done" should move the idea to Done tab and Clicking on "Later" should move the item to "Later" tab right? Also, clicking on either of them should remove the idea from "Idea" tab right?

**Comment by manparvesh** [comment]
@sangeethakp yes, you got that right. Also, after the idea is moved to `Later` or `Done`, the buttons should change accordingly.
